tests/ensure_provider_tests.py

```diff
@@ -114,11 +119,19 @@
 def create_nm_playbook(test_playbook):
     fileroot = os.path.splitext(os.path.basename(test_playbook))[0]
     nm_testfile = fileroot + "_nm.yml"
+    if NM_DEMANDING_TESTS.get(test_playbook, {}) != {}:
```

Comment:
```suggestion
    if test_playbook in NM_CONDITIONAL_TESTS:
```

Reply:
Changed into `if test_playbook in NM_CONDITIONAL_TESTS:`